test_scripts/Defects/7_0/3532_OnSCU_after_Show_main_IGN_OFF_ON_Cycle.lua

```diff
@@ -0,0 +1,87 @@
+---------------------------------------------------------------------------------------------------
+-- Issue: https://github.com/smartdevicelink/sdl_core/issues/3532
+--
+-- Description: Check that SDL transfer `OnSystemCapabilitiesUpdated` notification from HMI to an App
+-- after IGN_OFF/IGN_ON cycle in case App sent Show RPC with templateConfiguration parameter for main window
```

Comment:
Better to say `in next ignition cycle` or `after SDL restart` because IGN_OFF/IGN_ON cycle sounds to me not good.

Reply:
Comment is also applicable for the next script.